edge/README.md

```diff
@@ -0,0 +1,21 @@
+# Maximo Visual Inspection Edge
+
+This folder hierarchy contains resources for Maximo Visual Inspection Edge (MVIE).
+
+MVIE is designed to bring the power of AI models trained with MVI to the edge as a self-contained solution that can run on edge nodes_ anything from servers to laptops to devices such as an nVidia Jetson or a Raspberry Pi.
```

Comment:
extra underscore after "_nodes_" in the sequence "_nodes anything" maybe a double dash?

Reply:
> extra underscore after "_nodes_" in the sequence "_nodes anything" maybe a double dash?

😕 I've never used double dashes. When did that become a thing?